common/src/main/java/com/wynntils/screens/guides/WynntilsGuidesListScreen.java

```diff
@@ -68,6 +90,35 @@ protected void doInit() {
                 this));
     }
 
+    private void importFavorites() {
+        String clipboard = McUtils.mc().keyboardHandler.getClipboard();
+
+        if (clipboard == null || !clipboard.startsWith("wynntilsFavorites,")) {
+            McUtils.sendMessageToClient(Component.translatable("screens.wynntils.wynntilsGuides.invalidClipboard")
+                    .withStyle(ChatFormatting.RED));
+        }
+
+        ArrayList<String> names = new ArrayList<>(Arrays.asList(clipboard.split(",")));
+        names.remove(0); // Remove the "wynntilsFavorites," part
+        names.forEach(name -> {
+            if (name.isBlank() || name.isEmpty()) return;
+            Models.Favorites.addFavorite(name);
+        });
+        McUtils.sendMessageToClient(
+                Component.translatable("screens.wynntils.wynntilsGuides.importedFavorites", names.size())
+                        .withStyle(ChatFormatting.GREEN));
+    }
+
+    private void exportFavorites() {
+        McUtils.mc()
+                .keyboardHandler
+                .setClipboard("wynntilsFavorites" + String.join(",", Models.Favorites.getFavoriteItems()));
```

Comment:
Is this right? doesn't this produce "wynntilsFavoritesa,b" for [a, b]? (Does this work for legacy as well)

Reply:
Oh yeah my bad, forgot the comma. It's fine on legacy.